sdks/go/pkg/beam/runners/prism/internal/engine/strategy_test.go

```diff
@@ -44,3 +44,391 @@ func TestEarliestCompletion(t *testing.T) {
 		}
 	}
 }
+
+func TestTriggers_isReady(t *testing.T) {
```

Comment:
Aha I found the file: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/testing/data/trigger_transcripts.yaml

There are fewer cases in here that I would expect/hope. But this is where Robert started to gather them in an SDK-independent way.